_includes/2020/templates/Head.php

```diff
@@ -13,6 +13,9 @@ static function render($fields = []) {
       if(!isset($fields->title)) {
         $fields->title = 'Keyman | Type to the world in your language';
       }
+      if(!isset($fields->language)) {
```

Comment:
I think this gives us flexibility for a page to specify `$language='my';` in a header

Reply:
From other PRs, checking `empty()` better

```suggestion
      if(!empty($fields->language)) {
```